app/js/components/systemMetadata/systemMetadata.directive.js

```diff
@@ -128,6 +129,25 @@ function systemMetadataCtrl(
         return $scope.system.system_id;
     };
 
+    /**
+     * Finds the active tab by searching for the property
+     * in $scope.tabs that === true. Only one tab should be active
+     * at once.
+     *
+     * @return {String} activeTab the name of the active tab.
+     */
+    function getActiveTab() {
```

Comment:
given that only one tab can be active at once I am wondering whether it would be better instead of the tabs object simply use a single `currentTab` variable with values [`system`, `network`, `null`]. IMHO we would not lose anything and it would simplify e.g. this function.